src/Symfony/Component/Cache/Tests/Adapter/PredisAdapterTest.php

```diff
@@ -45,8 +47,9 @@ public function testCreateConnection()
             'read_write_timeout' => 0,
             'tcp_nodelay' => true,
             'database' => '1',
+            'conn_uid' => $connectionParameters['conn_uid'], // this value cannot be predicted
```

Comment:
Is the `conn_uid` really always present or does this depend on the Predis version? I mean the tests used to pass before.

Reply:
You're right, let's make it conditional